### PR TITLE
Log discovery queries that is above a static configurable threshold

### PR DESCRIPTION
**Pull request by dilipdevaraj-sfdc**

### CODE REVIEW

**Comment by dilipdevaraj-sfdc** [comment]
@nkunal 
Can you pls review

File: ArgusCore/src/main/java/com/salesforce/dva/argus/service/schema/CachedDiscoveryService.java [hunk 1]
```diff
@@ -129,10 +134,50 @@ private void _disposeExecutorService() {
 			queries.add(query);
 		}
 		
-		_logger.debug("Time to get matching queries in ms: " + (System.nanoTime() - start) / 1000000);
+		long timeToGetQueriesMillis = (System.nanoTime() - start) / 1000000;
```
Comment (nkunal) [inline ArgusCore/src/main/java/com/salesforce/dva/argus/service/schema/CachedDiscoveryService.java]: Do we want to log the es query time separately than what was received from cache ? perhaps , we could move the timeToGetQueriesMillis in the cache miss block and log it there ?

Comment (dilipdevaraj-sfdc) [reply ArgusCore/src/main/java/com/salesforce/dva/argus/service/schema/CachedDiscoveryService.java]: I would tend to think this is the net time taken before the customer sees the query being returned independent of whether it is from cache or from underlying discovery service (ES or Hbase or any other implementation). Hence I logged it here.

Comment (dilipdevaraj-sfdc) [reply ArgusCore/src/main/java/com/salesforce/dva/argus/service/schema/CachedDiscoveryService.java]: Moved the code to cache miss block.

**Pull request merged**